test/problematic_tests/impl_test.py

```diff
@@ -0,0 +1,58 @@
+import os
+import pytest
+import traceback
+import unittest
+
+from configparser import ConfigParser
+
+from GenomeFileUtil.GenomeFileUtilImpl import GenomeFileUtil
+
+
+def assert_exception_correct(got: Exception, expected: Exception):
+    """
+    Compare raised exception with expected exception.
+
+    Args:
+        got (Exception): Exception received
+        expected (Exception): Exception expected
+    """
+    err = "".join(traceback.TracebackException.from_exception(got).format())
+    assert got.args == expected.args, err
+    assert type(got) == type(expected)
+
+
+class ImplTest(unittest.TestCase):
```

Comment:
At some point we should switch to pytest and switch to functions from classes

Reply:
I first wrote impl_test.py in pytest, but it was not compatible with others.

Reply:
yeah, it'd probably require updating all the tests

Reply:
When do you think we should start switching? There are quite a few we need to update.

Reply:
Not for this project, we don't have enough time left and there's lots to do